.github/workflows/ci.yml

```diff
@@ -70,3 +70,26 @@ jobs:
       - name: Run test
         run: |
           bash examples/test.sh
+
+  benchmark:
```

Comment:
Now that we're adding new jobs to CI, maybe it's time to add something like the `checks-pass` in the bitcoin canister repo? We can then configure this repo to ensure that `checks-pass` passes before PRs are merged.

Reply:
Done.